doc/src/guides/logo.rst

```diff
@@ -0,0 +1,32 @@
+===========
+SymPy Logo
+===========
+
+We would like to make it easy for you to include the SymPy project identity in
+your next academic paper, course materials, or presentation.
+
+.. image::  ../logo/sympy.svg
+  :width: 600
+  :align: center
+  :alt: SymPy Logo
+
+You can find this high-resolution version of the SymPy logo
+`here <https://github.com/sympy/sympy/blob/master/doc/src/logo/sympy.svg>`_.
+
+If you would like to generate SymPy's collection of official logos yourself,
+you can do so by running:
+
+.. code-block:: none
+
+    $ cd doc
+
+    $ make logo # will be stored in the _build/logo subdirectory
```

Comment:
We should mention that this requires the `rsvg-convert` function. We can cross-link to the docs dependencies page for instructions on how to install it.

Reply:
Done.